asap/em_montage_qc/detect_montage_defects.py

```diff
@@ -163,6 +169,21 @@ def detect_stitching_gaps(render, prestitched_stack, poststitched_stack,
     session.close()
     return gap_tiles
 
+def detect_distortion(render, poststitched_stack, zvalue, threshold_cutoff=[0.005, 0.005], pool_size=20):
```

Comment:
```suggestion
def detect_distortion(render, poststitched_stack, zvalue, threshold_cutoff=(0.005, 0.005)):
```
`threshold_cutoff` can default to a tuple here, and there is no concurrency in this method.